src/oemof/tabular/facades/experimental/battery_electric_vehicle.py

```diff
@@ -453,3 +453,259 @@ def build_solph_components(self):
+            v2g=True,
+            loss_rate=self.loss_rate,
+            efficiency_mob_g2v=self.efficiency_mob_g2v,
+            efficiency_mob_v2g=self.efficiency_mob_v2g,
+            efficiency_sto_in=self.efficiency_sto_in,
+            efficiency_sto_out=self.efficiency_sto_out,
+            efficiency_mob_electrical=self.efficiency_mob_electrical,
+            pkm_conversion_rate=self.pkm_conversion_rate,
+            # expandable=, todo: ?
+            lifetime=self.lifetime,
+            # age=, todo: ?
+            # invest_c_rate=, todo: ?
+            bev_storage_capacity=self.bev_storage_capacity,
+            # bev_capacity=, todo:?
+            bev_invest_cost=self.bev_invest_cost,
+            fixed_costs=self.fixed_costs,
+            variable_costs=self.variable_costs,
+            # balanced=, todo:?
+            input_parameters=None,
+            # output_parameters=, todo: weglassen?
+            # max_charging_power=self.max_charging_power, ToDo: was ist das hier?
+        )
+
+        mobility_nodes.append(bev_controlled_v2g)
+
+        bev_inflex = Bev(
+            label="Inflex",
+            electricity_bus=self.electricity_bus,
+            commodity_bus=transport_commodity_bus,
+            charging_power=self.charging_power_inflex,
+            # charging_potential=, todo: ?
+            availability=self.availability,
+            storage_capacity=self.storage_capacity_inflex,
+            # initial_storage_capacity=, ToDo: weglassen?
+            min_storage_level=self.min_storage_level,
+            max_storage_level=self.max_storage_level,
+            # drive_power=self.drive_power, ToDo: wofür?
+            drive_consumption=self.drive_consumption,
+            v2g=False,
+            loss_rate=self.loss_rate,
+            efficiency_mob_g2v=self.efficiency_mob_g2v,
+            efficiency_mob_v2g=0,
+            efficiency_sto_in=self.efficiency_sto_in,
+            efficiency_sto_out=self.efficiency_sto_out,
+            efficiency_mob_electrical=self.efficiency_mob_electrical,
+            pkm_conversion_rate=self.pkm_conversion_rate,
+            # expandable=, todo: ?
+            lifetime=self.lifetime,
+            # age=, todo: ?
+            # invest_c_rate=, todo: ?
+            bev_storage_capacity=self.bev_storage_capacity,
+            # bev_capacity=, todo:?
+            bev_invest_cost=self.bev_invest_cost,
+            fixed_costs=self.fixed_costs,
+            variable_costs=self.variable_costs,
+            # balanced=, todo:?
+            input_parameters=self.input_parameters_inflex,
+            # output_parameters=, todo: weglassen?
+        )
+
+        mobility_nodes.append(bev_inflex)
+
+        pkm_demand = Load(  # todo: was geht hier?
+            label="pkm_demand",
+            type="Load",
+            carrier="pkm",
+            bus=transport_commodity_bus,
+            amount=400,
+            profile=[0, 1, 0],
+        )
+
+        mobility_nodes.append(pkm_demand)
```

Comment:
This will also happen outside of the metafacade as the demand might be for bev and other vehicle technologies together (gasoline/diesel)